pkg/operator/controller/status/controller.go

```diff
@@ -140,7 +140,14 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 
 	co.Status.Conditions = mergeConditions(co.Status.Conditions, computeOperatorAvailableCondition(allIngressesAvailable))
 	co.Status.Conditions = mergeConditions(co.Status.Conditions, computeOperatorProgressingCondition(allIngressesAvailable, oldStatus.Versions, co.Status.Versions, r.OperatorReleaseVersion, r.IngressControllerImage))
-	co.Status.Conditions = mergeConditions(co.Status.Conditions, computeOperatorDegradedCondition(state.IngressControllers))
+	co.Status.Conditions = mergeConditions(co.Status.Conditions, computeOperatorIngressDegradedCondition(state.IngressControllers))
+
+	authStateObj := &operatorv1.Authentication{ObjectMeta: metav1.ObjectMeta{Name: "cluster"}}
```

Comment:
Just use an empty struct, the client will marshal into the object and doesn't read the object to construct the get request

Reply:
maybe I don't understand the controller-runtime client, how it will know it should fetch the "cluster" instance of resource?

Reply:
You address the object to fetch using `types.NamespacedName`:
```go
 r.client.Get(context.TODO(), types.NamespacedName{Namespace: "namespace", Name: "name"}, obj)
```

For a cluster scoped resource, omit the `Namespace` field.